app/src/main/graphql/project.graphql

```diff
@@ -1,5 +1,5 @@
-query FetchProjects($first: Int = 15, $cursor: String, $sort: ProjectSort, $state: PublicProjectState,  $backed: Boolean, $recommended: Boolean, $categoryId: String, $starred: Boolean, $staffPicks: Boolean, $searchTerm: String) {
-  projects(first: $first, after:$cursor, sort:$sort, backed:$backed, recommended:$recommended, categoryId: $categoryId, starred:$starred,  state: $state, staffPicks: $staffPicks, term: $searchTerm) {
+query FetchProjects($first: Int = 15, $cursor: String, $sort: ProjectSort, $state: PublicProjectState,  $backed: Boolean, $recommended: Boolean, $categoryId: String, $starred: Boolean, $staffPicks: Boolean, $searchTerm: String, $raised: RaisedBuckets) {
```

Comment:
Added the new parameter already to have the `RaisedBuckets` model definition from graphQL available. Still this parameter is not sent from the VM, will come on https://kickstarter.atlassian.net/browse/MBL-2352